introduction/release-notes.adoc

```diff
@@ -65,7 +65,6 @@ Red Hat Business Automation VSCode Bundle
 * OpenShift Images
 
 == Unsupported, Community
-* DMN Runner
 * Embeddable editors
```

Comment:
We can put the Chrome extension here :)

Reply:
https://chrome.google.com/webstore/detail/bpmn-dmn-test-scenario-ed/mgkfehibfkdpjkfjbikpchpcfimepckf

Reply:
@yesamer done